src/translations/en.json

```diff
@@ -8,5 +8,57 @@
     "log_out": "Log out",
     "mailbox": "Mailbox",
     "shopping_list": "Shopping list"
+  },
+  "state": {
+    "on": "On",
+    "off": "Off",
+    "home": "Home",
+    "not_home": "Away",
+    "unknown": "Unknown",
+    "open": "Open",
+    "opening": "Opening",
+    "closed": "Closed",
+    "closing": "Closing",
+    "playing": "Playing",
+    "paused": "Paused",
+    "idle": "Idle",
+    "standby": "Standby",
+    "armed": "Armed",
```

Comment:
Does Lokalise support descriptions? Messages like this must be translated in context.

Reply:
Agreed. I didn't see any way to add a description in our JSON source file, but we could at the least add a comment to the key when we upload it. It does seem that they support extra data for other formats, so maybe this is something that could be added: https://docs.lokalise.co/article/SOrbByx8ZT-custom-attributes